bentoml/cli/deployment.py

```diff
@@ -213,3 +227,225 @@ def check_deployment_status(archive_path, platform, region, stage, api_name):
+    )
+    @click.option("--namespace", type=click.STRING, help="Deployment's namespace")
+    @click.option("--labels", type=click.STRING)
+    @click.option("--annotations", type=click.STRING)
+    @click.option('--region')
+    @click.option('--stage')
+    @click.option('--instance-type')
+    @click.option('--instance-count')
+    @click.option('--api-name')
+    @click.option('--kube-namespace')
+    @click.option('--replicas')
+    @click.option('--service-name')
+    @click.option('--service-type')
+    def apply(
+        bento_tag,
+        deployment_name,
+        platform,
+        namespace=None,
+        labels=None,
+        annotations=None,
+        region=None,
+        stage=None,
+        instance_type=None,
+        instance_count=None,
+        api_name=None,
+        kube_namespace=None,
+        replicas=None,
+        service_name=None,
+        service_type=None,
+    ):
+        bento_name, bento_verison = parse_bento_tag(bento_tag)
+        print(bento_name, bento_verison)
+        spec = DeploymentSpec(
+            bento_name=bento_name,
+            bento_verison=bento_verison,
+            operator=get_deployment_operator_type(platform),
+        )
+        if platform == 'aws_sagemaker':
+            spec.sagemaker_operator_config = DeploymentSpec.SageMakerOperatorConfig(
+                region=region,
+                instance_count=instance_count,
+                instance_type=instance_type,
+                api_name=api_name,
+            )
+        elif platform == 'aws_lambda':
+            spec.aws_lambda_operator_config = DeploymentSpec.AwsLambdaOperatorConfig(
+                region=region, stage=stage
+            )
+        elif platform == 'gcp_function':
+            spec.gcp_function_operator_config = DeploymentSpec.GcpFunctionOperatorConfig(
+                region=region, stage=stage
+            )
+        elif platform == 'kubernetes':
+            spec.kubernetes_operator_config = DeploymentSpec.KubernetesOperatorConfig(
+                kube_namespace=kube_namespace,
+                replicas=replicas,
+                service_name=service_name,
+                service_type=service_type,
+            )
+        else:
+            raise BentoMLDeploymentException(
+                "Custom deployment configuration isn't supported in the current version"
+            )
+
+        result = get_yatai_service().ApplyDeployment(
+            ApplyDeploymentRequest(
+                deployment=Deployment(
+                    namespace=namespace,
+                    name=deployment_name,
+                    annotations=parse_key_value_string(annotations),
+                    labels=parse_key_value_string(labels),
+                    spec=spec,
+                )
+            )
+        )
+        if result.status.status_code != Status.OK:
+            _echo('Apply deployment {} failed'.format(bento_name), CLI_COLOR_ERROR)
+            display_response_status_error(result.status)
```

Comment:
Let's put all the error output into one line? for example:
```
Apply deployment ABC failed: "Error ALREADY_EXISTS: Deployment named ABC already exists"
```
note that the error status code doesn't mean anything to the user